src/components/MovieDetails.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import '../css/MovieDetails.css';
+
+const MovieDetails = ({
```

Comment:
This is a good addition for us to keep these details from rendering in a 500 error situation in the movie view component.